### DISCUSSION TITLE
not installed

### DISCUSSION THREAD

**Discussion by Snake221982** [LingmoOS/Community#9] [Bugs Report/缺陷提交] [answered]
use the latest image: https://packages.lingmo.org/lingmo/release/hydrogen-202409081130/iso/LingmoOS-2.0-Hydrogen-Release-desktop-amd64.iso

The package manager was unable to make changes to the installed system. Command <pre>apt-get --purge -q -y remove live-boot live-boot-doc live-config live-config-doc live-config-systemd live-config-systemd live-tools lingmo-live</pre > returned error code 100.

**Comment by chun-awa** [comment, marked as answer]
currently we have some problems in LingmoOS Hydrogen livecd, use these commands to reinstall dpkg
```shell
sudo apt update
apt download dpkg
sudo dpkg -i --force-all dpkg*.deb
```
